playbooks/roles/configfmid/defaults/main.yml

```diff
@@ -68,3 +68,6 @@ zowe_external_domain_name:
 # optional, no default value
 # required for zD&T (external IP is different from internal IP)
 zowe_external_ip_address:
+
+# property added to instance.env for extender test
+zowe_extender_test_property: EXAMPLE_EXTENDER_PROPERTY=TEST123
```

Comment:
I'm thinking this makes FMID installation a little different comparing to convenience build. So if it's better to add some comments for this variable to not confuse people, like "This variable is for PTF installation verification purpose, not used by any extensions. You are safe to ignore." blah blah.

Reply:
Thanks for the suggestions, added the comment.